CMakeLists.txt

```diff
@@ -17,6 +17,8 @@ set(CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/CMake" ${CMAKE_MODULE_PATH})
 # set the project name
 project(velox)
 
+option(VELOX_BUILD_TESTING "Enable Velox tests" ON)
```

Comment:
could we add similar logic for duckdb, like I added in 

> https://github.com/facebookincubator/velox/pull/383

Reply:
And more generally, could you import the other changes from that PR into this, so I can safely abandon that one?

Reply:
> And more generally, could you import the other changes from that PR into this, so I can safely abandon that one?

I have imported your changes from #383